src/Info.ts

```diff
@@ -176,4 +199,40 @@ export default class Info {
         info.counter++;
         return info;
     }
+
+    private levenshteinDistance(a: string, b: string): number {
```

Comment:
Cmon stelar, this is not the node mentality. Get yourself a package instead! [fast-levenshtein](https://www.npmjs.com/package/fast-levenshtein) has typescript typings available at `@types/fast-levenshtein`.